packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -183,28 +255,15 @@ export class StarkDropdownComponent extends AbstractStarkUiComponent implements
 		this.logger.debug(componentName + ": component initialized");
 		this.optionsAreSimpleTypes = this.areSimpleTypes();
 
-		if (StarkFormUtil.isFormControl(this.dropdownFormControl)) {
-			this.formControl = this.dropdownFormControl;
-		} else {
-			const isDisabled: boolean = this.isDisabled || false;
-			this.formControl = new FormControl(
-				{
-					value: this.value,
-					disabled: isDisabled
-				},
-				this.required ? [Validators.required] : []
-			);
-
-			this.formControlSubscription = this.formControl.valueChanges
-				.pipe(
-					filter((value: any) => value !== null),
-					distinctUntilChanged()
-				)
-				.subscribe((value: any) => {
-					if (this.selectionChanged) {
-						this.selectionChanged.emit(value);
-					}
-				});
+		// tslint:disable-next-line:no-null-keyword
+		this.ngControl = this.injector.get<NgControl>(<Type<NgControl>>NgControl, <any>null);
+
+		if (this.ngControl !== null) {
+			this.ngControl.valueAccessor = this;
+		}
+
+		if (this.formField) {
+			this.formField._elementRef.nativeElement.classList.add("stark-dropdown-mat-form-field", `stark-${this.color}`);
```

Comment:
Could you `"mat-form-field-type-mat-select"` this fixes the issue with the ellipsis.